module-services/service-bluetooth/ServiceBluetooth.cpp

```diff
@@ -375,6 +376,8 @@ auto ServiceBluetooth::handle(message::bluetooth::ResponseAuthenticatePairCancel
 auto ServiceBluetooth::handle(BluetoothMessage *msg) -> std::shared_ptr<sys::Message>
 {
     LOG_INFO("Bluetooth request!");
+    resetTimeoutTimer();
```

Comment:
Timeout timer has to reset with every action on bluetooth, cause it could be turned off e.g. during scanning and it would be pretty confusing.